spec/components/provider_interface/find_candidates/candidate_invited_banner_component_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe ProviderInterface::FindCandidates::CandidateInvitedBannerComponent, type: :component do
+  describe '#render' do
```

Comment:
Could add one more test case -- when an invite exists but it isn't published. But it's not a hill i will die on.